server/src/main/kotlin/suwayomi/tachidesk/manga/impl/util/DirName.kt

```diff
@@ -31,21 +31,23 @@ private fun getMangaDir(mangaId: Int): String {
     return "$sourceDir/$mangaDir"
 }
 
-private fun getChapterDir(
+private fun getChapterDirs(
     mangaId: Int,
     chapterId: Int,
-): String {
+): List<String> {
     val chapterEntry = transaction { ChapterTable.select { ChapterTable.id eq chapterId }.first() }
 
-    val chapterDir =
-        SafePath.buildValidFilename(
-            when {
-                chapterEntry[ChapterTable.scanlator] != null -> "${chapterEntry[ChapterTable.scanlator]}_${chapterEntry[ChapterTable.name]}"
-                else -> chapterEntry[ChapterTable.name]
-            },
-        )
-
-    return getMangaDir(mangaId) + "/$chapterDir"
+    return buildList {
+        if (!chapterEntry[ChapterTable.scanlator].isNullOrEmpty()) {
```

Comment:
is there a difference in this or line 45?

Reply:
Hmm, previously I wanted to read downloads where there was an empty scanlator, but I fixed that in another PR. I'll fix that